docs/features/datastores/2_Using Datastores/2_creating-datastores.md

```diff
@@ -102,6 +102,26 @@ grid datastore create s3://ryft-public-sample-data/esRedditJson/ --name lightnin
 
 :::
 
+### Using the `--no-copy` option via the CLI
+
+Example: 
+`grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy`
+
+In certain cases, your s3 bucket may fit one (or both) of the following criteria: (1)
+the bucket is continually updating with new data which you want included in a Grid datastore (2) the bucket is
+particularly large (leading to long Datastore creation times). In these cases,
+you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
+prevent grid from making a copy of the dataset, speeding up datastore creation time
+significantly when working with large buckets. 
+
+:::info 
+
+When using this flag, you cannot remove files from your bucket. If you'd like to add files, please create a new version of the datastore after you've added files to your bucket. Please note that Grid does not currently support private S3 buckets. 
+
+If using this flag via the grid public cloud, then the source bucket should be in the  AWS `us-east-1` region or there will be significant latency when you attempt to access the datastore files in a run / session. 
```

Comment:
```suggestion
Example: 
`grid datastore create S3://ruff-public-sample-data/esRedditJson --no-copy`

In certain cases, your S3 bucket may fit one (or both) of the following criteria:

(1) the bucket is continually updating with new data which you want included in a Grid Datastore 
(2) the bucket is particularly large (leading to long Datastore creation times)

In these cases, you can pass the `--no-copy` flag to the `grid datastore create` command. This flag will
prevent Grid from making a copy of the dataset, which significantly speeds up Datastore creation time.

:::info 

When using this flag, you cannot remove files from your bucket. If you'd like to add files, please create a new version of the Datastore after you've added files to your bucket. 

Please note that Grid does not currently support private S3 buckets. 

If you are using this flag via the Grid public cloud, then the source bucket should be in the  AWS `us-east-1` region or there will be significant latency when you attempt to access the Datastore files in a Run or Session.
```

Reply:
NB: I am capitalizing all of the following words: Grid, Run, Session, Datastore - to my knowledge these are Grid-specific words (as opposed to eg 'dataset') and should be capitalized. Happy to iterate further, but this my logic behind what gets capitalized in the docs etc